.github/workflows/create-jira-version-awie.yml

```diff
@@ -0,0 +1,34 @@
+---
+name: release-web
```

Comment:
change this name (applies to other wf files too)

Reply:
@bdauria  would "oss" be better (if we consider this is at a release level): release-oss
or "module_name" better (if we consider them individually): release-<module_name>

Reply:
release-<module_name> was selected as a solution after discussion with @bdauria

Reply:
suggestion was applied